bot/handlers/connect_button.py

```diff
@@ -10,14 +12,29 @@
 async def connect_button(message: Message):
     user_id = message.from_user.id
     try:
-        key = await vpn_client.get_client_key(user_id)
-        if key:
-            text = f"<pre>{key}</pre>"
+        user = await get_user_by_telegram_id(user_id)
+        if user.expires_at and user.expires_at <= datetime.now(
+            timezone.utc
+        ):  # 1) действующий пользователь с истекшей подпиской
             await message.answer(
-                f"{text}\n 👆 Это ваш VPN ключ. Коснитесь, чтобы скопировать"
+                "Ваш доступ истёк. Пожалуйста, оплатите, чтобы продолжить пользоваться VPN"
             )
-        else:
+            return
+        key = await get_vpn_key(
```

Comment:
```suggestion
        vpn_key = await get_vpn_key(
```
В комментарии выше я имел ввиду название переменной `key` :-)